packages/zoe/test/swingsetTests/zoe/vat-alice.js

```diff
@@ -93,7 +84,8 @@ const build = async (E, log, zoe, issuers, payments, installations, timer) => {
     const option = await E(seat).makeCallOption();
     await E(bobP).doCoveredCall(option);
     const payout = await payoutP;
-    const [moolaPayout, simoleanPayout] = await Promise.all(payout);
+    const moolaPayout = await payout.Asset;
+    const simoleanPayout = await payout.Price;
```

Comment:
same as above. (Doesn't need a separate reply; I'm just marking the place if a change is needed.)